packages/pn-pa-webapp/public/locales/de/common.json

```diff
@@ -4,105 +4,106 @@
     "annulla filtro": "Filter entfernen",
     "indietro": "Zurück",
     "annulla": "Abbrechen",
-    "enter": "Besuchen",
+    "enter": "Zugreifen",
     "exit": "Abmelden",
-    "continue": "Weiter",
+    "continue": "Fortfahren",
     "send": "Senden",
     "go-to-home": "Zurück zur Startseite",
-    "go-to-login": "Zugriff"
+    "go-to-login": "Anmelden",
+    "close": "Schließen"
   },
   "menu": {
     "notifications": "Bescheide",
     "users": "Benutzer",
     "groups": "Gruppen",
     "api-key": "API-Schlüssel",
     "statistics": "Statistiken",
-    "app-status": "Status der Plattform",
-    "aria-label": "Plattform-Bescheide Hauptmenü",
-    "aria-label-inner": "Plattform-Bescheide Absender Navigation"
+    "app-status": "Plattformstatus",
+    "aria-label": "Plattform-Bescheide Hauptnavigation",
+    "aria-label-inner": "Plattform-Bescheide Navigation Absender"
   },
   "header": {
     "logout": "Abmelden",
     "pago-pa-link": "Website von PagoPA S.p.A.",
     "reserved-area": "Area Riservata",
-    "notification-platform": "SEND - Servizio Notifiche Digitali"
+    "notification-platform": "SEND - Dienst für digitale Bescheide"
```

Comment:
@AliceAzzolini non mi torna che il nome di send sia tradotto per intero, non dovrebbe rimanere in italiano?

Reply:
e fa bene a non tornati. Avevo chiesto di non tradurlo, nemmeno il descriptor. Lo aggiusti tu su tutte le lingue o chiediamo all'agenzia di farlo per noi e di inviarci di nuovo i file?

Reply:
@NicolaGiornetta puoi occupartene tu? Il nome della piattaforma deve rimanere in italiano, quindi "SEND - Servizio Notifiche Digitali".
Andrebbe fatto un check su tutte le lingue

Reply:
@ZorbaDimatteo va bene, me ne occupo io!